build.sh

```diff
@@ -77,4 +77,4 @@ export PATH="$DOTNET_INSTALL_DIR:$PATH"
 # Disable first run since we want to control all package sources
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-dotnet msbuild $REPOROOT/build/build.proj /m:1 /nologo /p:Configuration=$CONFIGURATION /p:Platform="$PLATFORM" "${args[@]}"
+dotnet msbuild $REPOROOT/build/build.proj /m:1 /nologo /flp:logfile=project.log;verbosity=diagnostic /p:Configuration=$CONFIGURATION /p:Platform="$PLATFORM" "${args[@]}"
```

Comment:
I think having the `;` here is causing problems.  Unix uses ; as a separator between commands.  At a minimum I think this needs to change to  `'/flp:logfile="project.log;verbosity=diagnostic"'` but it may be better just to drop this extra argument.